features/fixtures/shared/scenarios/Scenario.m

```diff
@@ -74,15 +74,11 @@ - (void)run {
 }
 
 - (void)startBugsnag {
-    [self waitForNetworkConnectivity];
+    // [self waitForNetworkConnectivity]; // Disabled for now because MR v4 does not listen on /
```

Comment:
Please reference PLAT-5827, which I've raised for MazeRunner to provide a suitable endpoint.  I've made a note of that ticket to reinstate this line.